src/components/PaginationV2/PaginationV2.js

```diff
@@ -338,32 +326,16 @@ export default class PaginationV2 extends Component {
           <button
             className={backButtonClasses}
             onClick={this.decrementPage}
-            aria-label={backwardText}
+            aria-label="previous page"
             disabled={backButtonDisabled}>
-            {componentsX ? (
-              <CaretLeft24 />
-            ) : (
-              <Icon
```

Comment:
`v9` need to keep the old icon?

Reply:
@asudoh i think this whole code block is already inside of a `return componentsX ? ( … )` statement

Reply:
Got it - Thank you for clarifying @lovemecomputer!